code/__defines/_compile_options.dm

```diff
@@ -11,6 +11,9 @@
 //#define ZASDBG	 	// Uncomment to turn on super detailed ZAS debugging that probably won't even compile.
 #define MULTIZAS		// Uncomment to turn on Multi-Z ZAS Support!
 
+// Movement Compile Options
+//#define CARDINAL_INPUT_ONLY // Uncomment to disable diagonal player movement (restore previous cardinal-moves-only behavior)

```

Comment:
I feel like this should be a config var rather than a compile time var.

Reply:
On reflection performance would take a hit. Not sure it's worth including a debug flag for it though.